dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -0,0 +1,2118 @@
+            }
+        }
+
+
+        list = itemService.getMetadata(publication, "dc", "contributor", "author", Item.ANY);
+
+        for (MetadataValue mdv : list) {
+            if (StringUtils.equals(mdv.getValue(), "plain text three")) {
+                assertEquals(4, mdv.getPlace());
+            }
+        }
+
+
+        getClient(adminToken).perform(get("/api/core/relationships/" + thirdRelationshipIdString))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("leftPlace", is(3)));
+
+    }
+
+    /**
+     * This test will simply add Relationships between Items with a useForPlace attribute set to false for the
+     * RelationshipType. We want to test that the Relationships that are created will still have their place
+     * attributes handled in a correct way
+     * @throws Exception
+     */
+    @Test
+    public void addRelationshipsNotUseForPlace() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+
+        Item author1 = ItemBuilder.createItem(context, col1)
+                                  .withTitle("Author1")
+                                  .withIssueDate("2017-10-17")
+                                  .withAuthor("Smith, Donald")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item author2 = ItemBuilder.createItem(context, col2)
+                                  .withTitle("Author2")
+                                  .withIssueDate("2016-02-13")
+                                  .withAuthor("Smith, Maria")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item author3 = ItemBuilder.createItem(context, col2)
+                                  .withTitle("Author3")
+                                  .withIssueDate("2016-02-13")
+                                  .withAuthor("Maybe, Maybe")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item orgUnit1 = ItemBuilder.createItem(context, col3)
+                                   .withTitle("OrgUnit1")
+                                
```

Comment:
as by discussion in the REST Contract PR, IMHO this PUT operation has no sense. In any case what should be checked is the effect of the update on the place for author1 assuming that there are other publications linked to him before and after the publication that is "moved" to author2

Reply:
@abollini : I'm starting to suspect your comments have been put on the wrong lines here (maybe there's a GitHub bug going on here).  The last three here don't make sense as they seem to be on empty lines.  Maybe GitHub is to blame here?  Could you recheck these comments...either they got put on the wrong lines by GitHub, or these changes were already updated.

Reply:
it is strange. If I look to the conversation tab I see it in the wrong place but it looks fine now in the files tab (so yes, it is a github bug).
The comment is about the putRelationshipAdminAccess test method

https://github.com/DSpace/DSpace/pull/2376/files#diff-61a3266941bcb44b91094e4d3f330ba6R1433

```
 getClient(token).perform(get("/api/core/relationships/" + id))
                        .andExpect(status().isOk())
                        .andExpect(jsonPath("$.rightId", is(author2.getID().toString())));
```

and I'm point to my previous comment here https://github.com/DSpace/DSpace/pull/2376/files#r279211896